### PR TITLE
cat: add Hindi translation

### PR DESCRIPTION
**Pull request by aditya955**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by kbdharun** [comment]
Hi, Thanks for your contribution. Seems like you selected the wrong base branch I have updated it to `main` now.

File: pages.hi/linux/cat.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# cat
+
+> फ़ाइलों को प्रिंट और जोड़ता है.
+> अधिक जानकारी: <https://www.gnu.org/software/coreutils/cat>.
+
+- फ़ाइल की सामग्री को मानक (standard) आउटपुट पर प्रिंट करें:
+
+`cat {{path/to/file}}`
```
Comment (kbdharun) [inline pages.hi/linux/cat.md:8]:
```suggestion
`cat {{फ़ाइल/का/पथ}}`
```

File: pages.hi/linux/cat.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`cat {{path/to/file1 path/to/file2 ...}} >> {{path/to/output_file}}`
+
+- फ़ाइल की सामग्री को अनबफर्ड ([u]nbuffered) मोड में आउटपुट फ़ाइल में कॉपी करें:
+
+`cat -u {{/dev/tty12}} > {{/dev/tty13}}`
+
+- फ़ाइल में `stdin` लिखें:
+
+`cat - > {{path/to/file}}`
```
Comment (kbdharun) [inline pages.hi/linux/cat.md:24]:
```suggestion
`cat - > {{फ़ाइल/का/पथ}}`
```

File: pages.hi/linux/cat.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- संख्या ([n]umber) सभी आउटपुट लाइनें:
+
+`cat -n {{path/to/file}}`
```
Comment (kbdharun) [inline pages.hi/linux/cat.md:28]:
```suggestion
`cat -n {{फ़ाइल/का/पथ}}`
```

File: pages.hi/linux/cat.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- गैर-मुद्रण योग्य और खाली स्थान वाले वर्ण प्रदर्शित करें (`M-` उपसर्ग के साथ यदि गैर-ASCII है):
+
+`cat -v -t -e {{path/to/file}}`
```
Comment (kbdharun) [inline pages.hi/linux/cat.md:32]:
```suggestion
`cat -v -t -e {{फ़ाइल/का/पथ}}`
```

File: pages.hi/linux/cat.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+> फ़ाइलों को प्रिंट और जोड़ता है.
+> अधिक जानकारी: <https://www.gnu.org/software/coreutils/cat>.
+
+- फ़ाइल की सामग्री को मानक (standard) आउटपुट पर प्रिंट करें:
+
+`cat {{path/to/file}}`
+
+- एक आउटपुट फ़ाइल में कई फ़ाइलों को सम्‍मिलित करें:
+
+`cat {{path/to/file1 path/to/file2 ...}} > {{path/to/output_file}}`
```
Comment (kbdharun) [inline pages.hi/linux/cat.md:12]:
```suggestion
`cat {{फ़ाइल1/का/पथ फ़ाइल2/का/पथ ...}} > {{आउटपुट_फ़ाइल/का/पथ}}`
```

File: pages.hi/linux/cat.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- आउटपुट फ़ाइल में कई फ़ाइलें जोड़ें:
+
+`cat {{path/to/file1 path/to/file2 ...}} >> {{path/to/output_file}}`
```
Comment (kbdharun) [inline pages.hi/linux/cat.md:16]:
```suggestion
`cat {{फ़ाइल1/का/पथ फ़ाइल2/का/पथ ...}} >> {{आउटपुट_फ़ाइल/का/पथ}}`
```

File: pages.hi/linux/cat.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# cat
+
+> फ़ाइलों को प्रिंट और जोड़ता है.
```
Comment (kbdharun) [inline pages.hi/linux/cat.md:3]:
```suggestion
> फ़ाइलों को प्रिंट और जोड़ता है।
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution, I have some suggestions for this page. 

- Regarding page descriptions (except the more information link line) we use native period (`।` in this case).
- We translate path placeholder for native speakers to better understand the examples (Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

**Tip**: In the `Files changed` tab you can group all suggestions to make a single commit.

**Comment by aditya955** [comment]
Made all the required changes!

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**